release/lts/arm32v7/meta.json

```diff
@@ -2,15 +2,20 @@
     "IsLinux" : true,
     "UseLinuxVersion": false,
     "PackageFormat": "powershell-${PS_VERSION}-linux-arm32.tar.gz",
-    "osVersion": "Ubuntu 18.04 ARM 32v7",
+    "osVersion": "Ubuntu 20.04 ARM 32v7",
     "SkipGssNtlmSspTests": true,
     "Base64EncodePackageUrl": true,
     "UseAcr": true,
     "tagTemplates": [
-        "lts-arm32v7-ubuntu-#shorttag#"
+        "#psversion#-ubuntu-#shorttag#-arm32-#fulltag#",
```

Comment:
fix duplicate tag
```suggestion
        "lts-#psversion#-ubuntu-#shorttag#-arm32-#fulltag#",
```

Reply:
I made this change and also added "preview" to start of preview tag name (so it now reads: `preview-#psversion#-ubuntu-#shorttag#-arm32-#fulltag#"`)